drivers/adc/ad463x/ad463x.h

```diff
@@ -43,9 +43,11 @@
 /***************************** Include Files **********************************/
 /******************************************************************************/
 #include <stdint.h>
-#include "spi_engine.h"
 #include "no_os_util.h"
+#if !defined(USE_STANDARD_SPI)
```

Comment:
In general we have #ifndef in our code. Basically, it is the same, but maybe is more concise.

Reply:
ok, will change.